src/Javis.jl

```diff
@@ -152,10 +159,10 @@ end
 """
     Rotation(r::Union{Float64, Symbol})
 
-Rotation as a transition from `r` to itself.
-Can be used as a general non-animating rotation.
+Rotation as a transition from 0.0 to `r` .
+Can be used as a short-hand.
 """
-Rotation(r::Union{Float64, Symbol}) = Rotation(r, r)
+Rotation(r::Union{Float64, Symbol}) = Rotation(0.0, r)
 
```

Comment:
Since we decided to only thoroughly document exported functions, could you add a line of an example invocation of this function? To show how it can be used ?